lm75bd/lm75bd.c

```diff
@@ -27,7 +27,20 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
+  uint8_t buf[1];
+  uint8_t buf_1[2];
+  uint8_t tempRegister = 0x00;
+  uint8_t numbytes = 1;
+  buf[0] = tempRegister;
```

Comment:
Most of this code can be simplified or isn't needed. You can just initialize the buffers at the top by add {0} and then you can remove the bottom 3 lines here